netbox/ipam/forms/filtersets.py

```diff
@@ -357,6 +357,15 @@ class IPAddressFilterForm(TenancyFilterForm, NetBoxModelFilterSetForm):
             choices=BOOLEAN_WITH_BLANK_CHOICES
         )
     )
+    dns_name = forms.CharField(
+        required=False,
+        widget=forms.TextInput(
+            attrs={
+                'placeholder': 'DNS Name',
+            }
+        ),
+        label='DNS Name'
```

Comment:
Please ensure human-readable labels are wrapped for translation.
```suggestion
        label=_('DNS Name')
```